src/content/developers/tutorials/transfers-and-approval-of-erc-20-tokens-from-a-solidity-smart-contract/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Transfers and approval of ERC-20 tokens from a solidity smart contract
 description: How to use a smart contract to interact with a token using the Solidity language
-author: "jdourlens"
+author: "jdourlens, updated 2022 by jcook"
```

Comment:
Unfortunately, our current design system doesn't support this. We should definitely explore this in our tutorial section revamp.

```suggestion
author: "jdourlens"
```